### PR TITLE
Feature/add frontend tools configs

### PR DESCRIPTION
**Pull request by aju**
Now tools configs are downloaded using npm.

Note:
Remember to run `./scripts/bootstrap.sh` and update your docker image as those config files are downloaded while building image!

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -214,3 +212,12 @@ gulp.task('watch', ['watch:sass', 'watch:docs-templates', 'watch:docs-sass']);
 gulp.task('build', function (done) {
     runSequence('clean:dist', 'sass:build', 'jekyll:docs', 'fingerprint', 'fingerprint-replace', 'docs:copy-components', 'sass:docs-build', done);
 });
+
+gulp.task('scss-lint', function() {
+    var scssLint = require('gulp-scss-lint');
+
+    return gulp.src(['src/**/*.scss', '!src/sass/vendors/**', '!src/docs/**', '!src/components/icons/_icons-data.scss'])
```
Comment (aju) [inline gulpfile.js]: We need it to exclude unwanted files

File: scripts/ide-scss-lint-exec.sh [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
-SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
-source ${SCRIPT_DIR}/run-scss-lint.sh $1 -f JSON
+# This script will run scss-lint
+
+PROJECT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd)
+
+docker run --rm \
+  -v $PROJECT_DIR/src:/style-guide/src \
+  -v $PROJECT_DIR/gulpfile.js:/style-guide/gulpfile.js \
+  brainly/style-guide scss-lint $1 --format JSON
```
Comment (aju) [inline scripts/ide-scss-lint-exec.sh]: We are not using gulp just plain scss-lint tool from console (just runing inside docker)

File: scripts/run-scss-lint.sh [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 PROJECT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd)
 
-docker run --rm \
-  -v $PROJECT_DIR:/style-guide/ \
-  brainly/style-guide scss-lint "$@"
+docker run -ti --rm \
+  -v $PROJECT_DIR/src:/style-guide/src \
```
Comment (aju) [inline scripts/run-scss-lint.sh]: Updated mounting - we were including node_modules before and it was causing errors!

File: scripts/run-scss-lint.sh [hunk 4]
```diff
@@ -4,6 +4,7 @@
...
+  -v $PROJECT_DIR/gulpfile.js:/style-guide/gulpfile.js \
+  brainly/style-guide gulp scss-lint
```
Comment (aju) [inline scripts/run-scss-lint.sh]: We simply run `gulp scss-lint` just from docker

File: scripts/ide-scss-lint-exec.sh (hunk 2 above)
Comment (mr-mig) [inline scripts/ide-scss-lint-exec.sh]: :+1:

**Comment by mr-mig** [comment]
:+1:

File: .gitignore [hunk 5]
```diff
@@ -1,4 +1,6 @@
 dist/
+.scss-lint.yml
+.editorconfig
 .idea
```
Comment (mr-mig) [inline .gitignore]: btw, it may be a good idea to put .gitignore inside the tools repo as well

Comment (aju) [inline .gitignore]: How can we then add project specific files there ? :(

**Comment by vergilius** [comment]
nice :+1:

**Pull request merged**